packages/product-details/src/ProductImages.js

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import ThumbnailSlider from './components/ThumbnailSlider'
+import SelectedImage from './components/SelectedImage'
+
+import VTEXClasses from './constants/productImagesClasses'
+import { VERTICAL, HORIZONTAL } from './constants/orientation'
+
+import './product-image.css'
+
+const DEFAULT_SELECTED_IMAGE = 0
+
+/**
+ *  Product Image component.
+ *  Display a list of thumbnail images in a slider and a main image of a product.
+ */
+class ProductImages extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      selectedImage: this.props.images[DEFAULT_SELECTED_IMAGE],
+    }
+  }
+
+  /**
+   * Function that changes the selected image
+   */
+  handleThumbnailClick = image => {
+    this.setState({
+      selectedImage: image,
+    })
+  }
+
+  render() {
+    const {
+      images,
+      thumbnailSliderOrientation,
+      thumbnailMaxVisibleItems,
+    } = this.props
+
+    const thumbnailProps = {
+      images,
+      onThumbnailClick: this.handleThumbnailClick,
+      orientation: thumbnailSliderOrientation,
+      maxVisibleItems: thumbnailMaxVisibleItems,
+    }
+
+    let className = `${VTEXClasses.MAIN_CLASS} flex inline-flex-ns`
+    if (thumbnailSliderOrientation === VERTICAL) {
+      className += ` ${VTEXClasses.VERTICAL_COMPONENT} `
+    } else {
+      className += ` ${VTEXClasses.HORIZONTAL_COMPONENT} flex-column-reverse`
```

Comment:
Could have used the `classnames` package

Reply:
Maybe a new issue... What do you think, @brunojdo ? I just moved the files and fixed minor things

Reply:
Yeah. This is another issue.